tests/test_common_view_logic.py

```diff
@@ -1,389 +1,182 @@
-                    "featured_media": "test",
-                    "group": "test_group",
-                    "image": "test_image",
-                    "tags": ["test"],
-                },
-                {
-                    "author": "test_author",
-                    "categories": [2, 3],
-                    "display_category": "test_category",
-                    "featured_media": "test2",
-                    "group": "test_group",
-                    "image": "test_image",
-                    "tags": ["test2"],
-                },
-            ],
-            "articles": [
-                {
-                    "author": "test_author",
-                    "categories": [1, 2],
-                    "display_category": "test_category",
-                    "featured_media": "test",
-                    "group": "test_group",
-                    "image": "test_image",
-                    "tags": ["test"],
-                },
-                {
-                    "author": "test_author",
-                    "categories": [2, 3],
-                    "display_category": "test_category",
-                    "featured_media": "test2",
-                    "group": "test_group",
-                    "image": "test_image",
-                    "tags": ["test2"],
-                },
-            ],
-            "groups": {1: "test_group"},
-            "upcoming": [],
-            "used_categories": {
-                1: "test_category",
-                2: "test_category",
-                3: "test_category",
-            },
-        }
-        self.maxDiff = None
-        self.assertEqual(context, expected_context)
-
-    @patch("canonicalwebteam.blog.wordpress_api.get_group_by_id")
-    @patch("canonicalwebteam.blog.wordpress_api.get_category_by_id")
-    @patch("canonicalwebteam.blog.wordpress_api.get_user")
-    @patch("canonicalwebteam.blog.wordpress_api.get_media")
-    def test_building_index_context_with_params_as_strings(
-        self, get_media, get_user, get_category_by_id, get_group_by_id
-    ):
-        get_media.return_value = "test_image"
-        get_user.return_value = "test_author"
-        get_category_by_id.return_value = "test_category"
-        get_group_by_id.return_value = "test_group"
-        articles = [
-            {
-                "featured_media": "test",
-                "author": "test",
-                "categories": [1, 2],
-                "group": [1],
-                "tags": ["test"],
-            },
-            {
-                "featured_media": "test2",
-                "author": "test2",
-                "categories": [2, 3],
-                "group": [1],
-                "tags": ["test2"],
-            },
+
+        featured, _ = api.get_articles(sticky=True)
+
+        # TODO: TDD refactor of function signature, to get
+        # the whole index without any params
+        index_context = get_index_context(1, articles, "2", featured)
+
+        # page information is correct
```

Comment:
please no comments. Same for the rest

Reply:
Im going to update them. But I do not agree with fully removing them, sorry.

Reply:
seems useless since code is quite self explanatory here. I am not confident with simple comments about their own maintenance.